### ISSUE TITLE
[BUG] The interface decorated with "@AiService" cannot be scanned by "AiServicesAutoConfig" when running the SpringBoot Jar package.

### ISSUE THREAD

**Issue by clear2x** [langchain4j/langchain4j#1030]
**Describe the bug**
The Spring Boot project packaged as a JAR file cannot scan the AiService annotated with `@AiService`.

**Log and Stack trace**

console output: 

```text
***************************
APPLICATION FAILED TO START
***************************

Description:

Parameter 0 of constructor in co.xx.XXXService required a bean of type 'co.xx.XXXAgent' that could not be found.
```

This is a screenshot of the debug output : 
![image](https://github.com/langchain4j/langchain4j/assets/40634471/bcd3c3b9-fdad-422f-a66d-75c11453766f)

The specific issue arises here:
![image](https://github.com/langchain4j/langchain4j/assets/40634471/a1352d21-ebba-400e-a443-6610a0a93b59)

**To Reproduce**
1. create a springboot project , use langchain4j : 
```xml
<dependency>
     <groupId>dev.langchain4j</groupId>
     <artifactId>langchain4j-spring-boot-starter</artifactId>
     <version>0.30.0</version>
</dependency>
<dependency>
     <groupId>dev.langchain4j</groupId>
     <artifactId>langchain4j-open-ai-spring-boot-starter</artifactId>
      <version>0.30.0</version>
</dependency>
```

2. Create an AiService and annotate it with `@AiService`.
3. mvn package & run  XXX.jar

**Expected behavior**

**Please complete the following information:**
- LangChain4j version: 0.30.0
- LLM(s) used:  OpenAI
- Java version:  Java8
- Spring Boot version (if applicable):  2.7.18

**Additional context**

**Comment by langchain4j** [comment]
Hi @clear2x thanks a lot for reporting!
Since you've debugged it already, do you see a way to fix it?
Thanks

**Comment by clear2x** [comment]
I attempted to fix this. Here's the PR :   https://github.com/langchain4j/langchain4j-spring/pull/20

**Comment by langchain4j** [comment]
@clear2x thank you a lot! I will check it asap

**Comment by suiquantong** [comment]
请问这个问题解决了嘛  为什么我还是启动不了

**Comment by langchain4j** [comment]
Fixed by https://github.com/langchain4j/langchain4j/issues/1290
Will be released in 0.32.0 today/tomorrow
